clouddriver-artifacts/src/main/java/com/netflix/spinnaker/clouddriver/artifacts/gcs/GcsArtifactCredentials.java

```diff
@@ -25,6 +25,7 @@
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.storage.Storage;
 import com.google.api.services.storage.StorageScopes;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Sort order is off here.

Reply:
Ah, java noob question: Which part of the import name should I be sorting on?  I've just been doing alphabetical, so "org" comes after "com"

Reply:
That's right, but you also have "com" entries below this.

Reply:
doh